apps/gimbtonomy/main.cpp

```diff
@@ -3,22 +3,51 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "ActuatorController.h"
 #include "CANMsg.h"
+#include "ContServo.h"
+#include "Encoder.h"
+#include "EncoderAbsolute_PWM.h"
+#include "LimServo.h"
 #include "Neopixel_Blocking.h"
 #include "can_config.h"
 #include "mbed.h"
+#include "mbed_config.h"
 #include "rover_config.h"
 
-CAN can(CAN1_RX, CAN1_TX, ROVER_CANBUS_FREQUENCY);
+#define POS_TOL   1
+#define MOVE_FLAG (1UL << 0)
+
+// Init. Components
+// Servos
+ContServo panServo(SRVO_PWM_CR, 44.0, 2.1, 0.9);  // 44 RPM at 4.8V, max->2100us PW, min->900us PW
+LimServo pitchServo(SRVO_PWM_MG, 180, 2.1, 0.9);
+
+Encoder::t_encoderConfig panEncoderConfig{ENC_PWM_GIMB, NC, NC, NC, NC, NC, NC, 360.0, 0, Encoder::x2_encoding};
+EncoderAbsolute_PWM panEncoder(panEncoderConfig);
```

Comment:
Ideally you would be able to use the servo class with the actuator class to use the PID and limit functions there. There was a ServoMotor class that was made to do this but it would need other parameters. (ServoMotor extended Motor, so it could be passed to the Actuator class just like Motor is).

Reply:
I see, I simply formatted this file from the original PR... could you explain why is this config for a relative encoder?